source/_integrations/roborock.markdown

```diff
@@ -112,6 +114,18 @@ Do not disturb - This enables _Do not disturb_ during the time frame you have se
 
 Volume - This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.
 
+### Button
+
+There are currently four buttons that allow you to reset the various maintenance items on your vacuum. Pressing the button cannot be undone. For this reason, the buttons are disabled by default to make sure they are not pressed unintentionally.
+
+Reset sensor consumable - The sensors on your vacuum are expected to be cleaned after 30 hours of use.
+
+Reset side brush consumable - The side brush is expected to be replaced every 200 hours.
+
+Reset main brush consumable - The main brush/ roller is expected to be replaced every 300 hours.
+
+Reset air filter - The air filter is expected to be replaced every 150 hours.
+
```

Comment:
just to be sure I understand this correctly: pressing the reset button means the consumable was replaced / cleaned?

Reply:
Yep!